keyboards/atlantis/ps17/info.json

```diff
@@ -0,0 +1,99 @@
+{
+    "manufacturer": "Atlantis",
+    "keyboard_name": "PS17",
+    "maintainer": "mjbogusz",
+    "url": "https://qmk.fm/keyboards/",
+    "processor": "atmega32u4",
+    "bootloader": "atmel-dfu",
+    "bootloader_instructions": "To reset the board into bootloader mode, tap the Reset switch mounted on the bottom of the PCB.",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x414B",
+        "vid": "0x0015"
+    },
+    "features": {
+        "bootmagic": false,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true,
```

Comment:
If this is enabled at the keyboard level, there should some encoder functionality in `ps17.c` as well. See [doc](https://docs.qmk.fm/#/feature_encoders?id=callbacks).